includes/Traits/Find_Readme.php

```diff
@@ -25,7 +25,7 @@ trait Find_Readme {
 	 */
 	protected function filter_files_for_readme( array $files, $plugin_relative_path ) {
 		// Find the readme file.
-		$readme_list = self::filter_files_by_regex( $files, '/readme\.(txt|md)$/i' );
+		$readme_list = preg_grep( '/readme\.(txt|md)$/i', $files );
```

Comment:
This change was added as `filter_files_by_regex()` is provided by [Abstract_File_Check](https://github.com/WordPress/plugin-check/blob/a24d647a3e48e6b8f5daa87fd9002f730c2c1e1f/includes/Checker/Checks/Abstract_File_Check.php#L114-L116) and moving the method from there to somewhere else didn't feel appropriate.

As the function is nothing but a wrapper for `preg_grep()` it's removal seems highly appropriate.